airflow/www/static/js/tree/dagRuns/Bar.jsx

```diff
@@ -34,53 +34,76 @@ import { callModalDag } from '../../dag';
 
 const DagRunBar = ({
   run, max, index, totalRuns, containerRef,
-}) => (
-  <Box position="relative">
-    <Flex
-      height="100px"
-      alignItems="flex-end"
-      justifyContent="center"
-      pb="2px"
-      px="3px"
-      onClick={() => {
-        callModalDag({ execution_date: run.executionDate, dag_id: run.dagId, run_id: run.runId });
-      }}
-    >
-      <Tooltip
-        label={<DagRunTooltip dagRun={run} />}
-        hasArrow
-        portalProps={{ containerRef }}
-        placement="top"
-        openDelay={100}
+}) => {
+  let highlightHeight = '100%';
+  if (containerRef && containerRef.current) {
+    const table = containerRef.current.getElementsByTagName('table')[0];
+    highlightHeight = table.offsetHeight - 53; // subtract the height of the duration axis labels
```

Comment:
Hmmm magic numbers. Is there some way we can not hard code the height of another element here?